File: app2/src/main/java/com/ssz/framejava/module/home/IHomeContract.java
package com.ssz.framejava.module.home;


import com.ssz.framejava.base.ui.mvp.func.BasePresenter;
import com.ssz.framejava.base.ui.mvp.func.BaseView;
import com.ssz.framejava.base.func.ISuccessListener;

import io.reactivex.disposables.Disposable;

/**
 * @author : zsp
 * time : 2019 10 2019/10/10 11:08
 */
public interface IHomeContract {

    interface IView extends BaseView {
        void success();
        void error();
    }

    interface IPresenter extends BasePresenter<IView> {
        Disposable login();
        Disposable login2(ISuccessListener<Boolean> listener);
    }
}

File: app/src/main/java/com/ssz/studydemo/ui/home/IHomeContract.kt
package com.ssz.studydemo.ui.home

import com.ssz.studydemo.base.ui.mvp.func.BasePresenter
import com.ssz.studydemo.base.ui.mvp.func.BaseView


/**
 * @author : zsp
 * time : 2019 09 2019/9/17 15:05
 */
interface IHomeContract {

    interface IHomeView : BaseView<IHomePresenter>

    interface IHomePresenter : BasePresenter
}